examples/general_settings/sample/reference.trial.inputs.json

```diff
@@ -6,22 +6,21 @@
     "indexfile": "/datasettestinputs/dataset/references/hg38/v0/Homo_sapiens_assembly38.fasta.fai"
   },
   
-  "sample_trial.last_reference":[
-    [
-      {
-        "name": "hg38_last_reference",
-       
-        "last_reference_bck": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.RY32.lastdb.bck",
-        "last_reference_des": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.RY32.lastdb.des",
-        "last_reference_prj": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.RY32.lastdb.prj",
-        "last_reference_sds": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.RY32.lastdb.sds",
-        "last_reference_ssp": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.RY32.lastdb.ssp",
-        "last_reference_suf": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.RY32.lastdb.suf",
-        "last_reference_tis": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.RY32.lastdb.tis"
-      }
-    ]
-  ],
+  "sample_trial.last_reference": {
+    "name": "hg38_last_reference",
+
+    "last_reference_bck": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.RY32.lastdb.bck",
+    "last_reference_des": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.RY32.lastdb.des",
+    "last_reference_prj": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.RY32.lastdb.prj",
+    "last_reference_sds": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.RY32.lastdb.sds",
+    "last_reference_ssp": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.RY32.lastdb.ssp",
+    "last_reference_suf": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.RY32.lastdb.suf",
+    "last_reference_tis": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.RY32.lastdb.tis"
+  },
 
   "sample_trial.ref_modimers": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.modimers.tsv.gz",
-  "sample_trial.movie_modimers": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.modimers.tsv.gz"
+  "sample_trial.movie_modimers": "/datasetpbhumanwgs/dataset/reference/human_GRCh38_no_alt_analysis_set.modimers.tsv.gz",
```

Comment:
Curious about why this file is listed twice.

Reply:
Oh not listed twice. Just that the line was changed to add a comma. So listing the difference in the whole line

Reply:
Wondering why there are two different keys pointing to the same file path.

Reply:
valid point. checking to see if both are actually used or if it was an artifact from previous iterations

Reply:
Looks good. I think its an artifact. movie_modimers is never used in sample.

Reply:
Turns out it's because kmer consistency check was never implemented in the WDL workflow, so none of the modimer outputs are used.  I think that's because CMKC didn't ask for this.  This was the only place that jellyfish was used, so we could potentially clip all of the jellyfish workflows out.



For reference:
https://github.com/PacificBiosciences/pb-human-wgs-workflow-snakemake/blob/main/rules/sample_kmer_consistency.smk

This is a pairwise comparison of the modimers.tsv files from each movie, after subtracting the reference modimers.tsv.  Essentially:

movie1 modimers - reference modimers -> non-reference movie1 modimers
movie2 modimers - reference modimers -> non-reference movie2 modimers

count(subtract(non-reference movie1 modimers, non-reference movie2 modimers)) + count(subtract(non-reference movie2 modimers, non-reference movie1 modimers)) -> count of unique modimers from pairwise comparison

We report a metric representing the proportion of total non-reference modimers that are unique to one movie.  If this count is above some threshold, it's likely that the two movies (SMRT Cells) were loaded with different samples.